packages/web/src/components/link/Link.module.css

```diff
@@ -9,11 +9,3 @@
   text-decoration: underline !important;
   text-decoration-color: var(--primary-light-2);
 }
-
```

Comment:
This caused some problems - verified badges don't want to flip path color when hovered, so expose this as a new prop in harmony `applyHoverStylesToInnerSvg`